tox/session.py

```diff
@@ -225,6 +225,7 @@ def _popen(self, args, cwd, stdout, stderr, env=None):
             env = os.environ.copy()
         return self.session.popen(args, shell=False, cwd=str(cwd),
                                   universal_newlines=True,
+                                  preexec_fn=os.setsid,
```

Comment:
AFAIK setsid does only make sense on UNIX - so this needs diferrentiation and different treatment on Windows.